drivers/backup/portworx/portworx.go

```diff
@@ -927,6 +927,50 @@ func (p *portworx) GetVolumeBackupIDs(
 	return volumeBackupIDs, nil
 }
 
+// ListBackupCRs lists all the Backup CRs present under given namespace
+func (p *portworx) ListBackupCRs(
+	ctx context.Context,
+	namespace string,
+	clusterObj *api.ClusterObject,
+	orgID string) ([]string, error) {
+
+	allBackupCRNames := make([]string, 0)
+	_, storkClient, err := getKubernetesInstance(clusterObj)
+
+	storkApplicationBackupCR, err := storkClient.ListApplicationBackups(namespace, metav1.ListOptions{})
```

Comment:
switch cluster and stork.Instance()? @mkoppal-px 

handle err from getKubernetesInstance?

Reply:
For this test case the restore and backup are happening on same cluster so cluster switch is not be required. Even if it is required, we can add that later on in helper methods in backup_helper.go